SU-SWS/decanter
# Deploy using grunt

# READY FOR REVIEW

# Summary
- Removes tracking of styleguide from git repo
- Adds grunt function `grunt deploy` 
- `grunt deploy` compiles, packages, and pushes the style guide to the github.io repository.

# Needed By (Date)
- Whenever

# Urgency
- Real low

# Steps to Test

1. Check out this branch
2. Run npm install
3. Run `grunt deploy`
4. Visit https://github.com/SU-SWS/decanter.github.io and review the last commit for a recent timestamp.
5. Wait a few minutes for the cache to clear and review http://decanter.stanford.edu

# Affected Projects or Products
- Decanter website.

# Associated Issues and/or People
-  ? Probably but I haven't looked. ?

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
